alpsqutip/scalarprod/orthogonalize.py

```diff
@@ -191,21 +192,36 @@ def orthogonalize_basis_cholesky(basis, sp: Callable, tol: float = 1.0e-5):
 
     # Compute the inverse Gram matrix for the given basis
     gram = gram_matrix(basis=local_basis, sp=sp)
-    cholesky_gram_matrix = cholesky(gram, lower=False)
-    linv_t = inv(cholesky_gram_matrix).transpose()
+
```

Comment:
Cholesky orthogonalization now checks that the basis of operators is linearly independent.